src/sequence/seq_crdt.rs

```diff
@@ -167,6 +174,18 @@ where
 
     /// Apply a remote data CRDT operation to this replica of the Sequence.
     pub fn apply_data_op(&mut self, op: CrdtDataOperation<A, Entry>) -> Result<()> {
+        // First check op is validly signed.
+        // Note: Perms for the op are checked at the upper Sequence layer.
+        match op.signature {
```

Comment:
Just in case you like this other way that I usually prefer:
```
let sig = op.signature.ok_or_else(|| Error::Unexpected("No signature on crdt op".to_string()))?;
...
op.source.verify(&sig, &bytes_to_verify)?;
...
```